httpie/plugins/builtin.py

```diff
@@ -56,3 +66,14 @@ def get_auth(
         password: str
     ) -> requests.auth.HTTPDigestAuth:
         return requests.auth.HTTPDigestAuth(username, password)
+
+
+class BearerAuthPlugin(BuiltinAuthPlugin):
+    name = 'Bearer HTTP Auth'
+    auth_type = 'bearer'
+    netrc_parse = False
+    auth_parse = False
+
+    # noinspection PyMethodOverriding
+    def get_auth(self, **kwargs) -> requests.auth.HTTPDigestAuth:
+        return HTTPBearerAuth(self.raw_auth)
```

Comment:
I wonder if it would make sense / be possible to enable `netrc_parse` / `prompt_password`

Reply:
`netrc` generally assumes the stuff is username/password (even though you can omit `username`, it just becomes an empty string), I am not sure if it is useful. If you feel like it, I can turn that back on with a test.